mcs/class/System.ServiceModel.Web/Test/System.ServiceModel.Web/WebOperationContextTest.cs

```diff
@@ -48,7 +48,7 @@ namespace MonoTests.System.ServiceModel.Web
 	public class WebOperationContextTest
 	{
 // MonoTouch/FullAOT does not support dynamic proxy code generation.
-#if !MONOTOUCH && !FULL_AOT_RUNTIME
+#if !MOBILE && !FULL_AOT_RUNTIME
```

Comment:
This should be different define, we can run this on ANDROID

Reply:
Well okay, in this case I'll have to introduce that new `DISABLE_SECURITY` conditional from my "Martin removes thing" PR here.